README.md

```diff
@@ -58,7 +58,7 @@ $ sudo yum install ./eosio-1.7.1-1.el7.x86_64.rpm
 ```
 #### Centos RPM Package Uninstall
 ```sh
-$ sudo yum remove eosio.cdt
+$ sudo yum remove eosio
 ```
```

Comment:
The Fedora RPM uninstall below has same issue.

Reply:
Good call I was ignoring that in develop since we were going to remove it anyways; need it here tho